ddmrp/models/stock_warehouse_orderpoint.py

```diff
@@ -51,14 +51,16 @@ class StockWarehouseOrderpoint(models.Model):
                  "product_uom.rounding", "red_override")
     def _compute_red_zone(self):
         for rec in self:
+            
```

Comment:
@hparfr @lreficent Maybe a _get_lead_time_factor() method should be a good approach ?

Reply:
I don't think it will make sense now. It can be refactored later when some new buffers will be added. (like time buffer :dancing_men: )

Reply:
Ok, that's what I've in mind :smiley: